config/locales/fr.yml

```diff
@@ -590,7 +590,7 @@ doc_auth.headers.underage: Condition d’âge non remplie
 doc_auth.headers.unreadable_id: Nous n’avons pas pu lire votre pièce d’identité
 doc_auth.headings.address: Mettre à jour votre adresse postale
 doc_auth.headings.back: Verso de votre permis de conduire ou de votre carte d’identité d’un État
-doc_auth.headings.capture_complete: Nous avons vérifié votre pièce d’identité
+doc_auth.headings.capture_complete: Nous avons vérifié votre pièce d'identité
```

Comment:
[nit] should this be a curly apostrophe to match other french translations

Reply:
I updated it. It looks like no changes to the French with that change